src/main/kotlin/com/exactpro/th2/lwdataprovider/grpc/GrpcUtils.kt

```diff
@@ -44,7 +42,7 @@ fun List<com.exactpro.th2.dataprovider.grpc.MessageSearchRequest.ResponseFormat>
                 BASE_64 -> list.add(ResponseFormat.BASE_64)
                 PARSED -> list.add(ResponseFormat.PARSED)
                 else -> {
-                    logger.error { "Unrecognized response format: $responseFormat" }
+                    throw IllegalArgumentException("Unrecognized response format")
```

Comment:
We are in Kotlin. There is a method `error` that throws the IllegalArgumentException. It is better to use the method.
Also, the Kotlin has its own IllegalArgumentException but you are using Java's exception